cfgov/cfgov/settings/base.py

```diff
@@ -579,6 +629,8 @@
     'www.gravatar.com',
 )
 
+WAGTAIL_GRAVATAR_PROVIDER_URL = '//www.gravatar.com/avatar'
+
```

Comment:
We could probably abandon Gravatar, can't we?

Reply:
I think so. I dropped it.